share/sequelize/seeders/20200920195019-EthereumContract.js

```diff
@@ -1,68 +0,0 @@
-'use strict';
```

Comment:
Did we intend to delete this file?

Reply:
yes, bcz it was obstructing the migrations . Do i need to replace that?

Reply:
i have replaced it